frontend/src/Onboarding/RedirectScreen.tsx

```diff
@@ -27,6 +29,33 @@ export const RedirectScreen: React.FC<Props> = ({ redirectUrl }) => {
     boolean | undefined
   >();
 
+  const calculateAcademicYear = (completedCourses: ScheduleCourse[]) => {
+    // sort the courses from the earliest to lastest semester
+    if (completedCourses && completedCourses.length != 0) {
+      const sortedCourses = JSON.parse(JSON.stringify(completedCourses)).sort(
+        (first: ScheduleCourse, second: ScheduleCourse) => {
+          if (Number(first.semester!) < Number(second.semester!)) {
+            return -1;
+          } else if (Number(second.semester!) < Number(first.semester!)) {
+            return 1;
+          } else {
+            return 0;
+          }
+        }
+      );
+
+      const earliestSemesterYear = sortedCourses[0].semester?.substring(0, 4);
```

Comment:
`sortedCourses[0].semester` contains the year and 2 other numbers after it like "201810" so I substring to get just the first 4